pantheon-bundle/src/main/java/com/redhat/pantheon/asciidoctor/AsciidoctorService.java

```diff
@@ -142,25 +151,55 @@ private String buildModule(Module base, ModuleVersion moduleVersion, Map<String,
         try (ResourceResolver serviceResourceResolver = serviceResourceResolverProvider.getServiceResourceResolver()) {
             moduleVersion = serviceResourceResolver.getResource(moduleVersion.getPath()).adaptTo(ModuleVersion.class);
 
+            // process product and version.
+            ProductVersion productVersion = null;
+            productVersion = moduleVersion.metadata().map(Metadata::productVersion)
+                    .map(t -> {
+                        try {
+                            return t.getReference();
+                        } catch (RepositoryException e) {
+                            return null;
+                        }
+                    })
+                    .get();
+            String productName = null;
+            if (productVersion != null) {
+                productName = productVersion.getProduct().getValueMap().get("name").toString();
```

Comment:
you should use the new method here
```suggestion
                productName = productVersion.getProduct().name().get();
```

Reply:
done